src/Services/Notification.vala

```diff
@@ -38,7 +38,7 @@ public class Notifications.Notification : Object {
     public AppInfo? app_info = null;
 
     public signal void closed ();
-    public signal bool time_changed (TimeSpan span);
+    public signal bool time_changed (DateTime span);
```

Comment:
Since there is a possible confusion between GLib.DateTime and Granite.DateTime, I suggest the namespace is included.